test_requirements.txt

```diff
@@ -1,5 +1,14 @@
-pytest <4.0
-pytest-describe <0.12
-testing.postgresql
-alembic
-alembic-verify
+importlib-metadata==0.23
+pytest==3.10.1
+pytest-describe==0.11.1
+py==1.8.0
+scramp==1.1.0
+testing.common.database==2.0.3
+testing.postgresql==1.3.0
+pg8000==1.13.2
+pluggy==0.13.0
+six==1.12.0
```

Comment:
Why do we need six? Aren't we py3 only at this point?

Reply:
six is a dependency of alembic-verify. Perhaps I need some guidance here: Do we freeze literally *everything*, as I have done here, or do we freeze only our top-level dependencies? If so, quite a bit of this can go.